src/assets/locales/de/main.json

```diff
@@ -1329,21 +1329,23 @@
         "roleUpdateError": "Error while updating roles",
         "encoding": "Encoding(select supported encoding)",
         "encodingPlaceholder": "Select supported encoding",
-        "noneOption": "Nicht notwendig (für die Integration ist kein technischer User notwendig)",
+        "noneOption": "Nicht notwendig (für die Integration ist kein technisches benutzerprofile notwendig)",
```

Comment:
```suggestion
        "noneOption": "Nicht notwendig (für die Integration ist kein technisches Benutzerprofil notwendig)",
```

Reply:
updated

Reply:
@manojava-gk not quite solved, please camelcase `Benutzerprofil`

Reply:
Kindly check now